comictalker/__init__.py

```diff
@@ -21,12 +23,15 @@
 ]
 
 
-def get_talkers(version: str, cache: pathlib.Path) -> dict[str, ComicTalker]:
+def get_talkers(
+    version: str, cache: pathlib.Path, local_plugins: Sequence[EntryPoint] = tuple()
+) -> dict[str, ComicTalker]:
     """Returns all comic talker instances"""
     talkers: dict[str, ComicTalker] = {}
     ct_version = parse(version)
 
-    for talker in entry_points(group="comictagger.talker"):
+    # A dict is used, last plugin wins
+    for talker in itertools.chain(entry_points(group="comictagger.talker"), local_plugins):
         try:
             talker_cls = talker.load()
```

Comment:
Going by this I would expect the plugin in the `plugins` dir to be the last and running through the loop it is but, the path from my test is the same as the first plugin. That is: I have `gcd` installed via pip and if I also put the `gcd` plugin in the `plugins` dir (with a change to the `about` text mark the difference), I can see `plugin_finder` finds the `gcd` in the `plugins` dir with the correct path but when it comes here later and does the `talker.load()` it uses the (or hasn't changed anything) dir from the first `gcd` found (the installed via pip). Hope that makes sense?

Reply:
They have the same import path, so it looks through sys.path and finds the installed one first

Reply:
So it might be a good idea in the future to load something better but for now they will have to use distinct import paths. So for gcd to override it you would need to used a different directory name and update the entrypoint.
Dependencies can be included by extracting the wheel into the plugin directory eg `plugins/gcd_talker/mokkari` would allow `import mokkari` to work. Note that bundled modules will take priority so on any real release the bundled mokkari would be used instead of the mokkari in a plugin folder

Reply:
The deps thing is something to put in the wiki, I will see about rounding it out now we're near a freeze.